### DISCUSSION TITLE
Searching for tags

### DISCUSSION THREAD

**Discussion by StoryHack** [jrnl-org/jrnl#1902] [General]
I am on windows, using jrnl 3.3. Whenever I try to search using tags:
`jrnl @whatever`
My default text editor (nano) opens up the same way as if I had only typed "jrnl", rather than returning a list of entries with the tag. When I use:
`jrnl -contains "@whatever"`
It returns the expected results. Is there something I'm doing wrong, or is there a setting I need to change, or maybe something else?

**Comment by micahellison** [comment]
Hi @StoryHack, it sounds like you're using Powershell, which reserves the `@` symbol for some [specific uses](https://stackoverflow.com/questions/363884/what-does-the-symbol-do-in-powershell).

There are a few ways to work around this:
* Use the Command Prompt instead of PowerShell to run jrnl. It treats the `@` symbol like normal text.
* Enter your tags with a different character than `@`. The number sign `#` works by default, or if you want to use a different character, you can add it to the `tagsymbols` value [in your config file](https://jrnl.sh/en/stable/reference-config-file/).
* Bypass the `@` symbol in PowerShell. Your second example (`jrnl -contains "@whatever"`) is actually doing this by wrapping `@whatever` in quotes. For your first example, you'd run `jrnl "@whatever"`. For less typing, you can also escape the `@` character in PowerShell by preceding it with a backtick (`` ` ``) like so: ``jrnl `@whatever``

**Reply by StoryHack** [reply]
That's what I needed, thank you!
